TicketBooking/Airport/AirportEnum.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace TicketBooking.Airports;
+
+public enum Airport
+{
+    QueenAliaInternationalAirport
```

Comment:
@WaleedJubeh I don't mind the `Airport` suffix being there. Most airports include it in their names: "Miami International Airport", "El Dorado International Airport", "Los Angeles International Airport", etc, etc. It is indeed redundant, but it is an okay redundancy since the enum is trying to store the airport entire name.

An alternative could be to use the IATA code instead ("MIA", "BOG", "LAX", "AMM"), but that kind of domain appliance is just outside the scope of the whole exercise.

Reply:
@raghad-ns 
`QueenAliaInternationalAirport = 1`